ogusa/scripts/execute.py

```diff
@@ -61,7 +61,8 @@ def runner(output_base, baseline_dir, test=False, time_path=True,
         test=test, baseline=baseline, guid=guid,
         tx_func_est_path=os.path.join(
             output_base,'TxFuncEst_{}.pkl'.format(guid)
-        )
+        ),
+        start_year=user_params['start_year']
     )
```

Comment:
And here `start_year` is passed as a key word argument to `get_parameters`.

Reply:
Not sure what you are asking here - I think how `start_year` is passed to the tax function estimation is fine here and am suggesting we do similar to pass it to `parameters.get_parameters()`.

Reply:
Right, but this is where `start_year` is passed to `parameters.get_parameters()`.

Reply:
Ok - yes, this is the change we need.  Thanks!

Reply:
Great, no problem.